youtube_dl/extractor/antenna.py

```diff
@@ -0,0 +1,115 @@
+        raise ExtractorError('{0} returned error: {1}'.format(self.IE_NAME,
+                                                              media_selection_error.id), expected=True)
+
+    def _extract_info_rss(self, video_id, webpage):
+        formats = []
+        jplayer_url = 'http://www.antenna.gr/templates/data/jplayer?d=m&cid='
+        desc_re = r'<description><!\[CDATA\[(.+?)\]\]></description>'
+        rss = self._download_webpage(jplayer_url + video_id, video_id)
+        video_url = self._search_regex(r'file="(.+?)"', rss, 'url')
+        if video_url == 'http://extranet.antenna.gr/flvsteaming/GR.flv':
+            raise AntennaIE.MediaSelectionError('Content not available outside Greece.')
+        formats.extend(self._extract_akamai_formats(video_url, video_id))
+        for video_format in formats:
+            if video_format.get('format_note') == 'Quality selection URL':
+                formats.remove(video_format)
+        if len(formats) == 0:
+            raise AntennaIE.MediaSelectionError('No formats available')
+        title = self._og_search_title(webpage).split(' | ')[-1] or self._search_regex(r'<title>(.+?)</title>', rss,
+                                                                                      'title')[9:-3]
+        thumbnail = self._og_search_thumbnail(webpage)
+        desc = self._og_search_description(webpage).split(' | ')[-1] or self._search_regex(desc_re, rss, 'description')
+
+        return {
+            'id': video_id,
+            'formats': formats,
+            'title': title,
+            'thumbnail': thumbnail,
+            'description': desc,
+        }
+
+    def _extract_info_json(self, video_id, webpage):
+        formats = []
+        json_url = 'http://www.antenna.gr/templates/data/jsonPlayer?d=m&cid='
+        meta = self._download_json(json_url + video_id, video_id)
+        manifest_url = meta.get('url')
+        if manifest_url == 'http://extranet.antenna.gr/flvsteaming/GR.mp4':
+            raise AntennaIE.MediaSelectionError('Content not available outside Greece.')
+        formats.extend(self._extract_akamai_formats(manifest_url, video_id))
+        for video_format in formats:
+            if video_format.get('format_note') == 'Quality selection URL':
+                formats.remove(video_format)
+        if len(formats) == 0:
+            raise AntennaIE.MediaSelectionError('No formats available')
+        title = meta.get('title') or self._og_search_title(webpage)
+        thumbnail = meta.get('thumb') or self._og_search_thumbnail(webpage)
+        desc = self._html_search_meta('description', webpage) or self._og_search_description(webpage)
+        return {
+            'id': video_id,
+            'formats': formats,
+            'title': title,
+            'thumbnail': thumbnail,
+            'description': desc.split(' | ')[-1],
+        }
+
+    def _real_extract(self, url):
+        extractors = [self._extract_info_json, self._extract_info_rss]
```

Comment:
Is there any point in this? Can you provide example URL that is only available via JSON and only via RSS?